decidim-sortitions/app/views/decidim/sortitions/admin/sortitions/show.html.erb

```diff
@@ -23,7 +23,7 @@
           <td><%= l sortition.request_timestamp, format: :long %></td>
           <td><%= sortition.dice %></td>
           <td><%= sortition.target_items %></td>
-          <td><%= sortition_category sortition %></td>
+          <td><%= sortition_taxonomies sortition %></td>
```

Comment:
Previously this was previously presented as: 
```
<%= **model**.taxonomies.map { |taxonomy| decidim_sanitize_translated(taxonomy.name) }.join(", ") %>
```
This is a consistency issue, which can reveal a design issue.

Reply:
here is different, it follows the same logic as previous categories, not having any means "all activated", that's why uses a helper for the extra information